### ISSUE TITLE
No longer server side friendly

### ISSUE THREAD

**Issue by goldensunliu** [nkbt/react-copy-to-clipboard#30]
since the update earlier today to `copy-to-clipboard` 3.0. The component is no longer universal or server sider renderable. with an error on rendering:
```
ReferenceError: navigator is not defined
```
since in `copy-to-clipboard` 3.0 navigator from client is assumed to be there. https://github.com/sudodoki/copy-to-clipboard/blob/master/index.js#L5

**Comment by lemonCMS** [comment]
I confirm

**Comment by nkbt** [comment]
Opened https://github.com/sudodoki/copy-to-clipboard/issues/39

Meanwhile as a quick workaround for server you can add somewhere before your app dependency injected:
```js
global.navigator = {};
```

Similar to: https://github.com/nkbt/react-copy-to-clipboard/commit/868e329518a91c251aeb3987514af1c13d35f849#diff-910eb6f57886ca16c136101fb1699231R14

**Comment by nkbt** [comment]
Published + react-copy-to-clipboard@4.2.1
